src/test/java/org/broadinstitute/hellbender/tools/spark/sv/RunMinimalBWAMEMTest.java

```diff
@@ -0,0 +1,91 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.utils.test.ArgumentsBuilder;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+
+/**
+ * Tests RunMinimalBWAMEM.
+ */
+public final class RunMinimalBWAMEMTest extends CommandLineProgramTest {
+
+    private static final File TEST_DATA_DIR = new File(getTestDataDir(), "spark/sv/RunMinimalBWAMEM");
+
+    @Test(groups="sv")
+    public void testSeparate() throws IOException {
+
+        try{
+            CMDLineProgramModule.checkIfProgramIsAvailableOnHost("bwa");
+        } catch(final IOException e){
+            System.err.println(e.getMessage());
+            return;
+        }
+
+        final ArgumentsBuilder args = new ArgumentsBuilder();
+        final File samOutput = boilerPlate(args);
+
+        // input arguments
+        final File input = new File(TEST_DATA_DIR, "input_1.fastq");
+        args.add("-" + StandardArgumentDefinitions.INPUT_SHORT_NAME);
+        args.add(input.getAbsolutePath());
+        final File secondInput = new File(TEST_DATA_DIR, "input_2.fastq");
+        args.add("-" + "I2");
+        args.add(secondInput.getAbsolutePath());
+
+        this.runCommandLine(args.getArgsArray());
+
+        BufferedReader br = new BufferedReader(new FileReader(samOutput));
```

Comment:
that output is a bam file. Can you compare this bam file to an expected bam file?